### ISSUE TITLE
Cannot read property 'owner' of undefined

### ISSUE THREAD

**Issue by iezer** [emberjs/ember-test-helpers#328]
@cibernox I'm trying to migrate from `ember-native-dom-helpers` to `ember-test-helpers` following this PR: https://github.com/cibernox/ember-native-dom-helpers/pull/144, but I'm getting an error `Cannot read property 'owner' of undefined` when I call `visit()`. I'm using ember-source 2.18.2 ember-cli-qunit 4.3.2. Do I need to add a call to `setupApplication` in my `module-for-acceptance` or something? Are there any public apps that use these helpers as an example? Thanks.

**Comment by cibernox** [comment]
That was an oversight on my side, I should have made the deprecation smarter. Right now you can only use helpers imported from `@ember/test-helpers` on the tests using the new API. I'll try to add a check so this deprecation only shows if you use the helpers from ember-native-dom-helpers in the new tests, but it doesn't complain if you use the old API.

**Comment by cibernox** [comment]
*tl;dr;* For now, either ignore the deprecation or try to update your tests to the new API. There is a very nice codemod that will do almost  all the work for you in seconds: https://github.com/rwjblue/ember-qunit-codemod

**Comment by iezer** [comment]
@cibernox Thanks for the info!
